side-configuration/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.4"
+
+services:
+  side-configuration:
+    build: .
```

Comment:
this sidecar needs to be used inside the `docker-compose.yml` of the service using it, adding an `image:` here would make it simpler as that service wouldn't need to rebuild the image. Doesn't require the image to be pushed, only to be built locally.